csharp_dotnetcore/12.NLP-With-LUIS/README.md

```diff
@@ -0,0 +1,60 @@
+﻿﻿This sample shows how to integrate LUIS to a bot with ASP.Net Core 2. 
+
+## To try this sample
+- Clone the repository
+```bash
+git clone https://github.com/Microsoft/botbuilder-dotnet.git
+```
+
+https://github.com/microsoft/botbuilder-tools/tree/master/packages/Ludown
+
+## Prerequisites
+### Set up Luis
+- - Navigate to [LUIS portal](https://www.luis.ai).
+- Click the `Sign in` button.
+- Click on `My Apps`.
+- Click on the `Import new app` button.
+- Click on the `Choose File` and select [LUIS-Reminders.json](LUIS-Reminders.json) from the `BotBuilder-Samples\csharp_dotnetcore\12.NLP-With-LUIS\CognitiveModels` folder.
+- Update [LuisBot.bot](LuisBot.bot) file with your AppId, SubscriptionKey, Region and Version. 
+    You can find this information under "Publish" tab for your LUIS application at [LUIS portal](https://www.luis.ai).  For example, for https://westus.api.cognitive.microsoft.com/luis/v2.0/apps/XXXXXXXXXXXXX?subscription-key=YYYYYYYYYYYY&verbose=true&timezoneOffset=0&q= 
+
+    - AppId = XXXXXXXXXXXXX
+    - SubscriptionKey = YYYYYYYYYYYY
+    - Region =  westus
+
+    The Version is listed on the page.
+- Update [LuisBot.bot](LuisBot.bot) file with your Authoring Key.  
+    You can find this under your user settings at [luis.ai](https://www.luis.ai).  Click on your name in the upper right hand corner of the portal, and click on the "Settings" menu option.
+NOTE: Once you publish your app on LUIS portal for the first time, it takes some time for the endpoint to become available, about 5 minutes of wait should be sufficient.
+### (Optional) Install LuDown
+- (Optional) Install the LUDown [here](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/Ludown) to help descrive language understanding components for your bot.
```

Comment:
Casing. 

In the -tools repo it's: "LUDown"

In header here it's "LuDown". 
In the text here it's "LUDown".